alcs-frontend/src/app/features/board/recon-create-card-dialog/recon-create-card-dialog.component.ts

```diff
@@ -149,10 +149,10 @@ export class ReconCreateCardDialogComponent implements OnInit {
       }
 
       await this.reconsiderationService.create(recon);
-
       this.dialogRef.close(true);
       this.toastService.showSuccessToast('Reconsideration card created');
     } finally {
+      await this.reconsiderationService.fetchByBoard(this.currentBoardCode);
```

Comment:
This seems like an odd fix, this component is already returning true when closing to indicate to its parent that the parent needs to refresh / fetch. What does this fix?

Reply:
It is a leftover of refactoring, I'll delete it